api/pkg/transformer/types/table/table.go

```diff
@@ -218,26 +218,19 @@ func (t *Table) Sort(sortRules []*spec.SortColumnRule) error {
 
 // UpdateColumnsRaw add or update existing column with values specified in columnValues map
 func (t *Table) UpdateColumnsRaw(columnValues map[string]interface{}) error {
-	origColumns := t.Columns()
+	df := t.DataFrame()
```

Comment:
Can you explain what happens here and why do you change and use t.DataFrame()? Is there any performance gain and benchmark that support it?

Reply:
the behaviour we intended is already available in the function Mutate, which works on the dataframe directly.
It will be redundant and complex to re-implement the same logic ourselves, and from what I see our own logic may be less efficient.

Reply:
Can you share benchmarking for this? Something like this: https://github.com/gojek/merlin/blob/main/api/pkg/transformer/symbol/time_bench_test.go#L68-L132